### ISSUE TITLE
Grid view filtering: don't send valueTo if it's not a range parameter

### ISSUE THREAD

**Issue by teosarca** [metasfresh/metasfresh-webui-frontend#314]
![image](https://cloud.githubusercontent.com/assets/1244701/23012772/77d93f08-f430-11e6-93f7-2ae9e736caea.png)

**Comment by teosarca** [comment]
NOTE: we also have a backend issue about filtering, which will be fixed in https://github.com/metasfresh/metasfresh-webui/issues/131

**Comment by metas-rc** [comment]
Results of IT1

THe valueTo problem still persists. Please check the attached screenshot.

![valuetoissue_persists](https://cloud.githubusercontent.com/assets/16001442/23070900/45de4efc-f535-11e6-8fb8-e817b79e5fe9.png)

**Comment by metas-rc** [comment]
Note to IT: 
Please, when testing this task again, repeat the test from https://github.com/metasfresh/metasfresh-webui/issues/131 too.

**Comment by damianprzygodzki** [comment]
Fixed, now only datarange widget should return valueTo. But it was impossible to test it for BPartners because there is exception on 123 window.

```
WidgetType's class is not compatible with LookupDescriptor's class WidgetType: ProcessButton WidgetType value class: class de.metas.ui.web.window.datatypes.LookupValue$StringLookupValue LookupDescriptor: (sql dump)
```

**Comment by metas-lc** [comment]
IT2
valueTo does not appear anymore in Business Partner when filtering for name. OK

Regression test for valueTo:
1. open a window that has filters for date range: e.g. Sales order
2. filter for date ordered: 01.12.2016 - 01.01.2017
=> valueTo is missing, and there is a value set on 30.11.2016 NOK
=> not applied correctly: you can see the orders from 01.12.2016 until 16.09.7016 NOK

![date_filter_s1](https://cloud.githubusercontent.com/assets/16001666/23168361/8e7c3552-f850-11e6-82a5-203a944e7302.png)

![date_filter_s2](https://cloud.githubusercontent.com/assets/16001666/23168363/92922a0c-f850-11e6-833b-db1a221d96de.png)

![date_filter_s3](https://cloud.githubusercontent.com/assets/16001666/23168368/96fd9bc6-f850-11e6-8c5a-0e11a90e4c98.png)


![date_filter_s4](https://cloud.githubusercontent.com/assets/16001666/23168372/98e78cb2-f850-11e6-98a5-5547b430fbee.png)

**Comment by teosarca** [comment]
@damianprzygodzki 
> WidgetType's class is not compatible with LookupDescriptor's class WidgetType: ProcessButton WidgetType value class: class de.metas.ui.web.window.datatypes.LookupValue$StringLookupValue LookupDescriptor: (sql dump)

shall be fixed now...

**Comment by damianprzygodzki** [comment]
@teosarca thank you, @metas-lc should be fixed by #389

**Comment by metas-rc** [comment]
Results of IT3

Tested on http://w101.metasfresh.com:8090/

* ValueTO no longer set for non-range parameters OK
* Value and ValueTO set for range parameter OK

Problems found with date ranges. Created 2 followups:
https://github.com/metasfresh/metasfresh-webui/issues/177
https://github.com/metasfresh/metasfresh-webui-frontend/issues/406
